src/observation/encodeMatcher/index.js

```diff
@@ -72,6 +72,5 @@ export default function encodeMatcher<Element: Model>(query: QueryDescription):
 
   invariant(!join.length, `Queries with joins can't be encoded into a matcher`)
 
-  // $FlowFixMe
```

Comment:
I think we should add `TODO` or some other mark in place of `$FlowFixMe` so we could later on check/fix those flow errors.

Reply:
Hm, don't you think `: any` is essentially the same as `$FlowFixMe`?

Reply:
Kinda, but there are situations that `:any` is used to "change" type like `const worker: Worker = (new LokiWorker: any)`.

Reply:
Right, but they're used interchangeably — in both cases we do this to lie to Flow because something is broken or we can't figure it out. We want to avoid `any` the same way as we avoid `$FlowFixMe`. I think the only practical difference is that `(: any)` applies to an expression, whereas $FlowFixMe applies to the whole line. No?